src/go/k8s/pkg/resources/configmap_test.go

```diff
@@ -104,7 +104,7 @@ func TestEnsureConfigMap_AdditionalConfig(t *testing.T) {
 			name:                    "Primitive object in additional configuration",
 			additionalConfiguration: map[string]string{"redpanda.transactional_id_expiration_ms": "25920000000"},
 			expectedStrings:         []string{"transactional_id_expiration_ms: 25920000000"},
-			expectedHash:            "0cb36f0be0d64032a61eb51a5d2985ea",
+			expectedHash:            "9df8ec83985d2c4f5584ea15fdac892b",
```

Comment:
Note: these are changing due to the two to four space change

Reply:
Not anymore.